frontend/public/components/custom-resource-definition.tsx

```diff
@@ -107,6 +123,68 @@ const Established: React.FC<{ crd: CustomResourceDefinitionKind }> = ({ crd }) =
   );
 };
 
+const EmptyVersionsMsg: React.FC<{}> = () => <EmptyBox label="CRD Versions" />;
+
+const CRDVersionsHeader = [
+  {
+    title: 'Name',
+    transforms: [sortable],
+  },
+  {
+    title: 'Served',
+    transforms: [sortable],
+  },
+  {
+    title: 'Storage',
+    transforms: [sortable],
+  },
+];
+
+const CRDVersionsRows = (versions: CRDVersion[]) =>
+  _.map(versions, (version: CRDVersion) => [
+    version.name,
+    version.served.toString(),
+    version.storage.toString(),
+  ]);
+
+const CRDVersionTable: React.FC<CRDVersionProps> = ({ versions }) => {
+  const [versionRows, setVersionRows] = React.useState<string[][]>(CRDVersionsRows(versions));
+  const [sortBy, setSortBy] = React.useState<PFSortState>({});
+
+  const onSort = (_event, index, direction) => {
+    let sortedRows;
+    if (index === 0) {
+      sortedRows = versionRows.sort((a, b) => crdVersionSort(a[index], b[index]));
+    } else {
+      sortedRows = versionRows.sort((a, b) =>
+        a[index] < b[index] ? -1 : a[index] > b[index] ? 1 : 0,
+      );
```

Comment:
You shouldn't need a custom sort function here.

```suggestion
      sortedRows = versionRows.sort();
```

Reply:
This won't work as expected. I had the same thought at first, but we need to compare the row value at the current sortBy column, not the entire row. See https://github.com/openshift/console/pull/5355/files#r426004094

Reply:
@spadgett as @TheRealJon notes, we need to get the value out of the row array using index in order to sort.